solidity/README.md

```diff
@@ -153,27 +153,15 @@ Here we also use an argument "-w" to set the workchain id and argument "--setkey
 The command generates **.boc** file with name \<\*-msg-init.boc\>.
 
 ### 3) Account preparation
-Out contract is going to store its code and data in the blockchain, but it costs money. So we must transfer some grams to our future contract address before deploying the contract. For transfer use one of the addresses obtained on the **step 1**. 
+Out contract is going to store its code and data in the blockchain, but it costs money. So we must transfer some grams to our future contract address before deploying the contract. Some of the ways to get test grams are described below in paragraph **Getting test grams**.
```

Comment:
The contract is going to store its code and data in the blockchain, but it costs money. So we must transfer some coins to the future contract address before deploying the contract. Some of the ways to get test coins are covered in **Getting test grams**.